client/src/styles/global.scss

```diff
@@ -417,3 +417,72 @@ $primary-color: #112f4e;
   margin-block-end: auto;
   font-style: italic;
 }
+
+/* this is used by J40MainGridContainer to show a blue background */
+.j40-main-grid-blue-bk {
+  background-color: $j40-blue-background-color;
+}
+
+/* index (about) page */
+.j40-about-page {
+  .j40-about-large-circle-graphics {
+    flex: 1 0 10%;
+    align-self: flex-start;
+  }
+
+  .j40-about-image-col {
+    max-width: 10.5rem;
+  }
+
+  .j40-about-section-divider {
+    max-width: 1px;
+    max-height: 100px;
+    border-left: 1px black solid;
+    border-top: 1px black solid;
+  }
+
+  .j40-about-cards.usa-card-group > .usa-card {
+    .usa-card__container {
```

Comment:
kudos for figuring out how to use the USWDS cards styles! These overrides seem like such a pain!

Reply:
Yeah, may not be the best approach? It's something we can refactor if we want.